eng/Directory.Build.Data.props

```diff
@@ -13,6 +13,8 @@
     <SignAssembly>true</SignAssembly>
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <DisableClientSdkAnalyzers>false</DisableClientSdkAnalyzers>
+    <DisableClientSdkAnalyzers Condition="'$(MSBuildProjectName)' == 'Azure.Base'">true</DisableClientSdkAnalyzers>
```

Comment:
I'd prefer that the projects set this property instead of listing the exceptions here.